jcache/remote/src/main/java/org/infinispan/jcache/remote/JCache.java

```diff
@@ -428,61 +428,6 @@ protected Object getCacheStatisticsMXBean() {
       return this;
    }
 
-   private JCache<K, V> checkNotNullOrNullKV(Map<? extends K, ? extends V> map, String name) {
-      checkNotNull((Object) map, name);
-      if (containsNullKey(map)) {
-         throw log.parameterMustNotContainNullKeys(name);
```

Comment:
Internationalization helps with the messages, and the messages can help distinguish which field is null. Since `Objects.requireNonNull` is called in the same statement, I don't know if a stacktrace would be enough to figure out which field is null. If stacktrace is not enough, I think we need an internationalized solution here.